udn-bgp.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 The Kube-burner Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocp
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kube-burner/kube-burner/pkg/measurements"
+	"github.com/kube-burner/kube-burner/pkg/workloads"
+	"github.com/spf13/cobra"
+)
+
+var additionalMeasurementFactoryMap = map[string]measurements.NewMeasurementFactory{
```

Comment:
Good to see this in action. Except for Raul's comments, now that our CI cluster supports UDN, would you mind adding a small CI test please?

Reply:
> would you mind adding a small CI test please?

Hope this helps:
https://excalidraw.com/#json=zeQHOJcN8JjZ836iTV4oh,StqYzzNY52TP39utKkJ1PA

Reply:
I find difficult to add a new CI test for this scenario, as it needs FRR to be running with privileged permissions in the host running kube-burner. And if I understood correctly, this test needs kube-burner to be running in a host  in the same subnet of the OCP cluster in order to ping the routes.

Reply:
Yes, My idea is to get it added in prow just like other tests we have with special hardware requirements.